server/src/test/java/io/crate/integrationtests/UserSessionIntegrationTest.java

```diff
@@ -23,39 +23,12 @@
 
 import static io.crate.testing.Asserts.assertThat;
 
-import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.test.IntegTestCase;
 import org.junit.Test;
 
-import io.crate.execution.engine.collect.stats.JobsLogService;
-
 @IntegTestCase.ClusterScope(numDataNodes = 2, numClientNodes = 0, supportsDedicatedMasters = false)
 public class UserSessionIntegrationTest extends BaseRolesIntegrationTest {
 
-    @Override
-    protected Settings nodeSettings(int nodeOrdinal) {
-        Settings settings = super.nodeSettings(nodeOrdinal);
-        if (nodeOrdinal == 0) { // Enterprise enabled
-            return Settings.builder().put(settings)
-                .put(JobsLogService.STATS_ENABLED_SETTING.getKey(), true).build();
-        }
-        // Enterprise disabled
-        return Settings.builder().put(settings)
-            .put(JobsLogService.STATS_ENABLED_SETTING.getKey(), true).build();
-    }
-
-    @Test
-    public void testSystemExecutorUsesSuperuserSession() {
-        systemExecute("select username from sys.jobs", "sys", getNodeByEnterpriseNode(true));
-        assertThat(response).hasRows("crate");
-    }
-
-    @Test
-    public void testSystemExecutorNullUser() {
-        systemExecute("select username from sys.jobs", "sys", getNodeByEnterpriseNode(false));
-        assertThat(response).hasRows("crate");
-    }
```

Comment:
Initially I wanted to keep one test but then realized that `systemExecute` is pretty much same with `execute` nowadays (runs from `crate` superuser).

Test  `test_set_session_user_from_auth_superuser_to_unprivileged_user_round_trip` below  basically confirms that execute has crate session:
```
 try (var session = sqlExecutor.newSession()) {
            execute("SELECT SESSION_USER", session);
            assertThat(response).hasRows("crate");
```

so I don't think we need those tests.